tests/SiteSummaryExtensionTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use BringYourOwnIdeas\SecurityChecker\Tests\Stubs\SiteSummaryAlertStub;
+
+class SiteSummaryExtensionTest extends SapphireTest
+{
+    protected static $fixture_file = 'PackagesWithAlerts.yml';
+
+    protected $requiredExtensions = [
+        SiteSummary::class => [SiteSummaryAlertStub::class]
+    ];
+
+    public function setUp()
+    {
+        parent::setUp();
+        $manifest = new SS_ConfigManifest(BASE_PATH, true);
+        $moduleDoesNotExist = $manifest->moduleExists('bringyourownideas/silverstripe-maintenance');
+        if ($moduleDoesNotExist) {
+            $this->markTestSkipped(
+                'Module bringyourownideas/silverstripe-maintenance is required for this test, but is not present.'
+            );
+        }
+    }
+
+    public function testDisplayColumnsDoesNotIncludePrintingColumns()
+    {
+        $report = Injector::inst()->create('SiteSummary');
+        $this->assertArrayNotHasKey('listSecurityAlertIdentifiers', $report->columns());
+    }
+
+    public function testUpdateAlerts()
+    {
+        // The themes should not affect the test.
+        // Ensure we use the default template supplied with this module.
+        Config::inst()->update('SSViewer', 'theme_enabled', false);
+        
+        $report = Injector::inst()->create('SiteSummary');
+        $fields = $report->getCMSFields();
+        $alertSummary = $fields->fieldByName('AlertSummary');
+        $this->assertInstanceOf('LiteralField', $alertSummary);
+        $content = $alertSummary->getContent();
+        $this->assertContains(
+            'Sound the alarm!',
+            $content,
+            'ensure our extension doesn\'t override all others'
+        );
+        // Content is translatable, check for the identifying class instead.
```

Comment:
Yes but we don't normally check for translated text in any unit tests. If you want to you could add `_t()` calls into the tests (which is done occasionally) otherwise it's not worth mentioning this. Essentially here you're testing that the class is rendered into the template, and that's fine

Reply:
So ultimately you're saying I should remove this comment?

Reply:
I guess so 😆